docs/dev/Pagination-v2.md

```diff
@@ -54,29 +59,35 @@ Similarly to v1 engine, the response object is the same as initial response if t
 
 `cursor_id` will be different with each request.
 
-If this is the last page, the `cursor` property is ommitted. The cursor is closed automatically.
+## End of scrolling/paging
+
+When scrolling is finished, SQL plugin still returns a cursor. This cursor leads to the final page, which as no cursor and no data. Receiving that page means all data was properly queried and returned to user. Cursor is closed automatically on processing that page.
 
 The client will receive an [error response](#error-response) if executing this request results in an OpenSearch or SQL plug-in error.
 
 ## Cursor Keep Alive Timeout
-Each cursor has a keep alive timer associated with it. When the timer runs out, the cursor is closed by OpenSearch.
+
+Each cursor has a keep alive timer associated with it. When the timer runs out, the cursor is automatically closed by OpenSearch.
 
 This timer is reset every time a page is retrieved.
 
 The client will receive an [error response](#error-response) if it sends a cursor request for an expired cursor.
 
+Keep alive timeout is [configurable](../user/admin/settings.rst#plugins.sql.cursor.keep_alive) by setting `plugins.sql.cursor.keep_alive` and has default value of 1 minute.
+
 ## Error Response
+
 The client will receive an error response if any of the above REST calls result in an server-side error.
 
 The response object has the following format:
 ```json
 {
     "error": {
-        "details": <string>,
-        "reason": <string>,
-        "type": <string>
+        "details": "<string>",
+        "reason": "<string>",
+        "type": "<string>"
     },
-    "status": <integer>
+    "status": "<integer>"
```

Comment:
I know this satisfies the json format, but this looks wrong

Reply:
Fixed in d3888ceaf